client/app/pages/alert/index.js

```diff
@@ -11,6 +11,7 @@ function AlertCtrl($routeParams, $location, $sce, toastr, currentUser, Query, Ev
   this.trustAsHtml = html => $sce.trustAsHtml(html);
 
   this.onQuerySelected = (item) => {
+    this.alert.query = item;
```

Comment:
Right way to do it?

Reply:
It's Angular code, so it's okay. But if this function will be called from React component - ensure you call `$scope.$applyAsync()`

Reply:
It does but everything works ok 🤷‍♂️

Reply:
I think it works because you use Angular services later, and they trigger $digest at some point (I think, when `$q` promise resolves). But IMO to be 100% sure - it's better to explicitly call $applyAsync (because if that's promise - then data update will be delayed, and sometimes it may be noticeable).

Reply:
Done.